gen/src/index.ts

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+const yargs = require('yargs');
+const mkdirp = require('mkdirp');
+const fs = require('fs');
+const sqrl = require('squirrelly');
+
+const qt = require('qt');
+
+let IN = yargs.argv.in || process.env.IN;
+let OUT = yargs.argv.out || process.env.OUT;
+
+const LANGUAGE = 'python';
+const SRC_DIRECTORY = 'src';
+const TEMPLATE_DIRECTORY = 'templates';
+const PY_FILENAME = 'events.py';
+const INIT_PY_TEMPLATE = 'init_py.squirrelly';
+const README_TEMPLATE = 'README.squirrelly';
+const SETUP_PY_TEMPLATE = 'setup_py.squirrelly';
+const VERSION = '0.0.1';
+
+async function main() {
+  if (!IN) console.error('Error in config: `IN` not set');
+  if (!OUT) console.error('Error in config: `OUT` not set');
+  if (!IN || !OUT) return;
+  if (IN.endsWith('/')) IN = IN.substring(0, IN.length - 1);
+  if (OUT.endsWith('/')) OUT = OUT.substring(0, OUT.length - 1);
+
+  const templateDirectoryPath = `${__dirname}/../../${TEMPLATE_DIRECTORY}`;
+
+  const schemasAndGenFiles = await qt.getJSONSchemasAndGenFiles(IN, LANGUAGE);
+  const allEvents: Array<[string, string, string]> = [];
+  schemasAndGenFiles.map((sg: [any, string]) => {
+    const schema = sg[0];
+    const genFile = sg[1];
```

Comment:
nit: We can use array destructuring to make this cleaner:

```ts
schemasAndGenFiles.map(([schema, genFile]) => {
```

Reply:
Done.